### PR TITLE
Wired up GAM page with Campaign and Seller information

### PR DESCRIPTION
**Pull request by danthaman44**
Test plan
Screen Shot 2020-08-07 at 10 02 32 PM

### CODE REVIEW

File: src/components/MerchantsPage/gam/CampaignListItem.tsx [hunk 1]
```diff
@@ -5,47 +5,84 @@ import {
...
         <ImagesContainer>
-          <img src={apexLogo} alt="merchant_logo" />
-          <img src={melonpannaLogo} alt="distributor_logo" />
+          {distributor &&
+            (<a href={distributor.website_url}>
+              <img src={distributor.image_url} alt="distributor_logo" />
+            </a>
+            )
+          }
+          <img src={melonpannaLogo} alt="merchant_logo" />
```
Comment (danthaman44) [inline src/components/MerchantsPage/gam/CampaignListItem.tsx:75]: Need to expose a "logo_image_url" field on the Seller model, the current hero_image_url and owner_image_url can't be used for this

File: src/components/MerchantsPage/gam/CampaignListItem.tsx [hunk 2]
```diff
@@ -5,47 +5,84 @@ import {
...
         </ImagesContainer>
-        <Button className="button--filled">Visit merchant</Button>
+        {merchant && (
+          <Button className="button--filled" onClick={(e) => {
+            e.preventDefault();
+            window.location.href = '/' + merchant.seller_id;
+          }}>Visit merchant</Button>
+        )}
         <Button className={'button--outlined'}>Gift a meal</Button>
```
Comment (danthaman44) [inline src/components/MerchantsPage/gam/CampaignListItem.tsx:83]: Still need to wire this up to the GAM flow @oliviasztanga

File: src/components/MerchantsPage/gam/CampaignListItem.tsx [hunk 3]
```diff
@@ -5,47 +5,85 @@ import {
...
-          isActive={true}
-          numContributions={73}
-          targetAmount={100}
+          isActive={campaign.active}
+          numContributions={mealsRaised}
+          targetAmount={targetMeals}
           progressBarColor={'#CF6E8A'}
-          lastContributionTime={new Date('07/21/2020 20:05:00')}
-          endDate={new Date('07/23/2020')}
+          lastContributionTime={new Date(campaign.last_contribution)}
```
Comment (justintmckibben) [inline src/components/MerchantsPage/gam/CampaignListItem.tsx:64]: We don't want this to be X hours ago?

Comment (danthaman44) [reply src/components/MerchantsPage/gam/CampaignListItem.tsx:64]: It changes to 'X' hours ago if the lastContributionTime is recent enough! And even 'X' minutes ago if the time was less than 1 hour ago.

**Pull request merged**